### PR TITLE
resolve all methods which contained deprecations

### PR DESCRIPTION
**Pull request by foxriver76**
Resolving the deprecations in controller code, so we can make a release in the near future

### CODE REVIEW

File: packages/controller/lib/setup/setupUpgrade.js [hunk 1]
```diff
@@ -149,13 +125,13 @@ function Upgrade(options) {
 
             debug(`upgrade order is "${sortedAdapters.join(', ')}"`);
 
-            await this._upgradeAdapterHelper(repo, sortedAdapters, 0, forceDowngrade, autoConfirm);
```
Comment (foxriver76) [inline packages/controller/lib/setup/setupUpgrade.js:152]: this method was no longer existing

**Review by Apollon77 — APPROVED** [review]
LGTM

File: packages/controller/lib/setup/setupUpgrade.js (hunk 1 above)
Comment (GermanBluefox) [reply packages/controller/lib/setup/setupUpgrade.js:152]: Yes. It was  replaced by "for" cycle and no more required

**Comment by GermanBluefox** [comment]
LGTM

**Pull request merged**